hazelcast/src/main/java/com/hazelcast/scheduledexecutor/impl/ScheduledTaskDescriptor.java

```diff
@@ -242,14 +235,8 @@ public int hashCode() {
 
     @Override
     public String toString() {
-        return "ScheduledTaskDescriptor{"
-                + "definition=" + definition + ", "
-                + "future=" + future + ", "
-                + "stats=" + stats + ", "
-                + "state=" + state + ", "
-                + "isTaskOwner=" + isTaskOwner + ", "
-                + "result=" + resultRef.get()
-                + '}';
+        return "ScheduledTaskDescriptor{" + "definition=" + definition + ", " + "future=" + future + ", " + "stats=" + stats
```

Comment:
I would prefer the old style, which I find more readable. It's also easier for refactorings (complete lines are added, removed or re-ordered).

If it should stay collapsed, I would remove the superfluous string concats to improve readability. Same for the follwing `toString()` methods.

Reply:
Tried to make the template + formatter to work hand-in-hand to produce this automatically, but I failed. Maybe after 3.10 I can try again, too much time spent on it. :/